airflow/www_rbac/views.py

```diff
@@ -564,7 +568,8 @@ def dag_details(self, session=None):
         dag_id = request.args.get('dag_id')
         dag_orm = DagModel.get_dagmodel(dag_id, session=session)
         # FIXME: items needed for this view should move to the database
-        dag = dag_orm.get_dag(STORE_SERIALIZED_DAGS)
+        dag = dag_orm.get_dag(
+            conf.getboolean('core', 'store_serialized_dags', fallback=False))
```

Comment:
No. This does not make sense.

Thsi change just makes it slower. STORE_SERIALIZED_DAGS is `conf.getboolean('core', 'store_serialized_dags', fallback=False))`, but without calling the slow conf.getboolean every time.

Reply:
Sure. Airflow does not support dynamic settings. And even if it did, it should not be one of them.